host/xtest/xtest_main.c

```diff
@@ -150,6 +154,12 @@ int main(int argc, char *argv[])
 
 	opterr = 0;
 
+	if (argc > 1 && !strcmp(argv[1], "sha-perf")){
+		/* Run sha-perf */
+		sha_perf_runner_cmd_parser(argc-1, &argv[1]);
```

Comment:
And for convenience, compare also `argv[0].` Something like this maybe? (untested).
```
char *c;

c = strrchr(argv[0], '/');
if (c)
        c++;
else
        c = &argv[0];
if (!strcmp(c, "sha-perf")
        return sha_perf_runner_cmd_parser(argc, argv);
```